src/public/js/cardModal.js

```diff
@@ -0,0 +1,73 @@
+(function($) {
+    //this file should be imported into the board handlebars file
+    //this variable should be the <div> that the modal is going to be inserted in to
+    let placeModalHere = $("#placeModalHere");
```

Comment:
look at my board html and let me know if what I have does not work with how you plan for this to work, I can change it to add another div with some id="modal-container" or something like that